client/src/redux/project/project.types.ts

```diff
@@ -1,15 +1,21 @@
 export interface Project {
-	_id: string;
+	_id?: string;
 	title: string;
 	description: string;
-	image: string;
+	image?: string;
```

Comment:
Note: adding an image field to the create project dialog can be implemented later, leave a note in the code for this